lib/gcp/src/cloudSQL.ts

```diff
@@ -107,22 +107,36 @@ export function* ensureCloudSQLExists({
   // To save us some errors in our logs, just wait a bit here.
   // Would be nice to find a clean way to wait for the peering to be complete
   // but I couldn't find anything to help me with that.
-  yield delay(10 * SECOND);
+  let attemptNumber = 0;
+  const sqlInstanceCreationDeadline = Date.now() + 2 * 60 * SECOND; // 2 mins.
 
   log.info(`Ensure SQLInstance exists`);
-  const existingInstance: sql_v1beta4.Schema$DatabaseInstance = yield call(
-    ensureSQLInstanceExists,
-    { instance, opstraceClusterName }
-  );
 
-  if (!existingInstance.name) {
-    throw Error("SQLInstance did not return a name");
-  }
+  while (sqlInstanceCreationDeadline > Date.now()) {
+    log.info(`Attempt ${attemptNumber++} to create SQLInstance`);
+    yield delay(10 * SECOND);
```

Comment:
Pragmatic local retry :+1: 

In the longer run, we should for GCP have the same conceptual approach as for AWS, with the `AWSResource` clas, implementing the same common looping/retrying logic for both, with the only difference being error handling code.

Reply:
Yes! 👍